Desktop 4.2 docs updates (#13776)
docker/docker.github.io: desktop/windows/index.md

### Software Updates

The **Software Updates** section notifies you of any updates available to Docker Desktop. When there's a new update, you can choose to download the update right away, or click the **Release Notes** option to learn what's included in the updated version.

> **Disable automatic check for updates**
>

> > Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal. Update to [Docker Desktop 4.2.0](../windows/release-notes/index.md) to start using this feature.

Reviewer: ```suggestion
> Starting with Docker Desktop 4.2.0, the option to turn off the automatic check for updates is available for users on all Docker subscriptions, including Docker Personal and Docker Pro. Update to [Docker Desktop 4.2.0](../windows/release-notes/index.md) to start using this feature.
```

Author: Updated

{: .important}

Turn off the check for updates by clearing the **Automatically Check for Updates** check box. This disables notifications in the Docker menu and also the notification badge that appears on the Docker Dashboard. To check for updates manually, select the **Check for updates** option in the Docker menu.

> To allow Docker Desktop to automatically download new updates in the background, select **Always download updates**. This downloads newer versions of Docker Desktop when an update becomes available.

Reviewer: ```suggestion
To allow Docker Desktop to automatically download new updates in the background, select **Always download updates**. This downloads newer versions of Docker Desktop when an update becomes available. You will still have to apply and restart the update once it is downloaded. You can do this either through the Docker Menu or the Updates Section in the Docker Dashboard
```

Author: Updated

Select **Use nightly builds** to download nightly builds instead of stable builds When you check for available updates.

After selecting your software update preferences, click **Apply and Restart** for the changes to take effect.

## Dashboard

> The Docker Desktop Dashboard enables you to interact with containers and applications and manage the lifecycle of your applications directly from your machine. The Dashboard UI shows all running, stopped, and started containers with their state. It provides an intuitive interface to perform common actions to inspect and manage containers and existing Docker Compose applications. For more information, see [Docker Desktop Dashboard](../dashboard.md).

Reviewer 2: nit: is this not the `Docker Dashboard`?
```suggestion
The Docker Dashboard enables you to interact with containers and applications and manage the life-cycle of your applications directly from your machine. The Dashboard UI shows all running, stopped, and started containers with their state. It provides an intuitive interface to perform common actions to inspect and manage containers and existing Docker Compose applications. For more information, see [Docker Dashboard](../dashboard.md).
```

Reviewer 3: Yes, it's Docker Dashboard, but lifecycle is one word.

Author: Good catch. Updated to Docker Dashboard
